### ISSUE TITLE
Only show inputs that are available based on the license status (Don't show licensed inputs on a cluster without a license)

### ISSUE THREAD

**Issue by drewmiranda-gl** [Graylog2/graylog2-server#14176]
## What?

While doing research on Graylog inputs, what is offered, and which are licensed or not, I noticed that even if I remove my license from the graylog cluster, all licensed inputs are still visible and theres no indication if an input is licensed or not. Only after you create the input and attempt to start it do you get any indication that the input is licensed.

![image](https://user-images.githubusercontent.com/107503402/206734621-dcae8b97-c6a8-4587-8053-0ae237d781ca.png)
![image](https://user-images.githubusercontent.com/107503402/206734677-5efa9a32-c58d-4455-a171-e0ac9b7474d2.png)
![image](https://user-images.githubusercontent.com/107503402/206734705-57f8a1f0-5f36-417e-a0ca-94d96daf5ebf.png)
![image](https://user-images.githubusercontent.com/107503402/206734742-7e06fe66-6e9c-4652-a608-329b500b6797.png)

I understand these inputs may not normally appear as long as the integrations plugins are NOT installed. However there could be cases where this is installed but the cluster is not licensed. It would much more clear if the input is licensed or not (similar to how the list of alert notification types show)

![image](https://user-images.githubusercontent.com/107503402/206734905-5fa5ff0e-ae5a-49ea-a99f-f9fbdc4f933d.png)

## Why?

Currently its quite difficult to understand which inputs are part of Graylog Open Source and which are paid/licensed. It would help the experience to both label which are licensed and not show licensed inputs if no valid license is present.

## Your Environment

* Graylog Version: 5

**Comment by drewmiranda-gl** [comment]
stale, won't fix
